tests/helpers.js

```diff
@@ -1,16 +1,11 @@
 import path from "path";
-import fs from "fs";
 import devices from "puppeteer/DeviceDescriptors";
 
 export function getStorybookUrl() {
-  let location = path.join(__dirname, "..", "storybook-static");
+  const location = path.join(__dirname, "..", "storybook-static");
+  console.log({ location });
 
-  if (fs.existsSync(location)) {
-    location = `file://${location}`;
-  } else {
-    location = "http://localhost:3006";
-  }
-  return location;
+  return `file://${location}`;
```

Comment:
I think this change makes sense. We should only support one method of snapshotting that is identical everywhere